webpack.common.config.js

```diff
@@ -24,7 +24,8 @@ module.exports = function (outputFilename) {
         'okta/underscore': SHARED_JS + '/util/underscore-wrapper',
         'okta/handlebars': 'handlebars/dist/handlebars',
         'okta/moment': 'moment/moment',
-        'okta/moment-tz': VENDOR + '/lib/moment-timezone-with-data-2010-2020',
+        'okta/moment-tz': EMPTY,
```

Comment:
@mauriciocastillosilva-okta quick search suggests we're not using this, but can you confirm?

Reply:
yeah, I don't see it either. And I can't think of a UI where we display times... errors like 'we sent you an sms in the last minute' or 'do not challenge me for N minutes on this device' are all not using moment as far as I know.